### ISSUE TITLE
bug: unmarshalling error in `victoria-logs-single`

### ISSUE THREAD

**Issue by dhess** [VictoriaMetrics/helm-charts#1795]
**Chart name and version**
chart: victoria-logs-single
version: v0.8.2

**Describe the bug**

When both Grafana dashboard support and Grafana operator support are enabled, Helm complains about an unmarshalling error.

I believe this is new since `v0.8.2`. The same `values.yaml` was working fine previously.

```
error: map[string]interface {}(nil): yaml: unmarshal errors:
  line 9: mapping key "app" already defined at line 8
```

**Custom values**
Please provide only custom values (excluding default ones):
```yaml
dashboards:
  enabled: true
  grafanaOperator:
    enabled: true

```

**Comment by AndrewChubatiuk** [comment]
hey @dhess 
thanks for an issue
expect a fix in the next release soon

**Comment by AndrewChubatiuk** [comment]
available in 0.8.3

**Comment by dhess** [comment]
Confirmed working, thanks!
